Source/AwsCommonRuntimeKit/mqtt/MqttRequestResponseClient.swift

```diff
@@ -327,58 +461,58 @@ internal class MqttRequestResponseClientCore: @unchecked Sendable {
         }
     }
     
-    /// create a stream operation, throws CRTError if the creation failed. You would need call open() on the operation to start the stream
-    public func createStream(streamOptions: StreamingOperationOptions) throws -> StreamingOperation {
-        // TODO: create streamming operation
-        return StreamingOperation()
+    /// Creates a stream operation, throws CRTError if the creation failed. You would need call open() on the operation to start the stream
+    fileprivate func createStream(streamOptions: StreamingOperationOptions) throws -> StreamingOperation {
+        return try StreamingOperation(streamOptions: streamOptions, client: self)
     }
     
-    /// release the request response client. You must not use the client after call `close()`.
-    public func close() {
+    /// Releases the request response client. You must not use the client after call `close()`.
```

Comment:
Trivial: "You must not use the client after call `close()`." "You must not use the client after calling `close()`."